src/operations/save_breeze_pipe.cr

```diff
@@ -0,0 +1,6 @@
+class SaveBreezePipe < BreezePipe::SaveOperation
```

Comment:
To Save a BreezePipe you'd need to add a Pulsar event in Lucky when a pipe runs. Send the name of it, whether it is before/after, etc.

Then you can hook subscribe to the event like we do for queries https://github.com/paulcsmith/breeze-poc-2/pull/1/files#diff-ea2779fd001ab38c934f10bb3a7674c039754b6d222790cf16d1c8302d220080R14